doc(release): Prepare release notes for oss 23.04 (#2628)
centreon/centreon-documentation: versioned_docs/version-23.04/releases/centreon-os.mdx

## Centreon Web

### 23.04.8

> Release date: `September 21, 2023`

Reviewer: ```suggestion
Release date: `September 22, 2023`
```

<details open>
  <summary>Enhancements</summary>

- [Core] Added feature flag system.
- [Configuration] Added feature flag to vault configuration endpoints.
- [Authentication] Removed password fields from profile page for accounts using delegated authentication.
- [Anomaly Detection] Added a feature flag to hide the period exclusion interface.

</details>

> <details open>

Reviewer: ```suggestion
<details>
```

  <summary>Bug fixes</summary>

- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
- [CLAPI] Fixed the export of severities to make it possible to reimport them.
- [CLAPI] Removed the possibility to link a regular resource with a template.	
- [Authentication] Fixed access to the Centreon UI when a default page is configured for the user and authentication uses SSO (OIDC/SAML).	
- [Security] Fixed SLQi in centreonLogAction.class.php.
- [Resources Status] Fixed filters on hostgroups and categories that weren't returning hosts.
- [Configuration] Fixed custom macro inheritance for hosts.

> 

Reviewer: ```suggestion

</details>

```

## Centreon Web

### 23.04.8

Release date: `September 21, 2023`

<details open>
  <summary>Enhancements</summary>

> - [Core] Added feature flag system

Reviewer: ```suggestion
- [Core] Added feature flag system.
```

> - [Configuration] Added feature flag to Vault Configuration endpoints.

Reviewer: ```suggestion
- [Configuration] Added feature flag to vault configuration endpoints.
```

> - [authentication] Removed password fields if contact is not local type.	

Reviewer: ```suggestion
- [Authentication] Removed password fields from profile page for accounts using delegated authentication.
```

> - [Anomaly Detection] Added a feature flag to hide exclusion of periods.

Reviewer: ```suggestion
- [Anomaly Detection] Added a feature flag to hide the period exclusion interface.
```

</details>

<details open>
  <summary>Bug fixes</summary>

- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
- [Anomaly Detection] Don't list Business Activities as service in configuration form.
- [CLAPI] Fixed the export of severities to be able to reimport them.
- [CLAPI] Removed the possibility to link a regular resource with a template.	
- [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).	

> - Fixed SLQi in centreonLogAction.class.php.

Reviewer 2: ```suggestion
- [Security] Fixed SLQi in centreonLogAction.class.php.
```

> - [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
> - [Anomaly Detection] Don't list Business Activities as service in configuration form.
> - [CLAPI] Fixed the export of severities to be able to reimport them.
> - [CLAPI] Removed the possibility to link a regular resource with a template.	
> - [Authentication] Fixed the access to Centreon UI when a default page is defined using SSO authentication (OIDC/SAML).	
> - [Security] Fixed SLQi in centreonLogAction.class.php.
> - [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
> - [Resources Status] Fixed filter on hostgroups and categories that doesn't returning hosts.
> - [Configuration] Fixed custom macro inheritance for hosts.
> 

Reviewer: ```suggestion
- [CLAPI] Fixed a PHP warning that was displayed when using getparam on a service.
- [Anomaly Detection] Business Activities are no longer listed as services in the configuration form.
- [CLAPI] Fixed the export of severities to make it possible to reimport them.
- [CLAPI] Removed the possibility to link a regular resource with a template.	
- [Authentication] Fixed access to the Centreon UI when a default page is configured for the user and authentication uses SSO (OIDC/SAML).	
- [Security] Fixed SLQi in centreonLogAction.class.php.
- [Resources Status] Fixed filters on hostgroups and categories that weren't returning hosts.
- [Configuration] Fixed custom macro inheritance for hosts.

```
